### PR TITLE
Add support for new "org.jetbrains.kotlin.js" plugin.

### LINKED ISSUE #252: Add support for new JS Gradle plugin
With Kotlin release `1.3.40` new JS Gradle plugin was [introduced](https://blog.jetbrains.com/kotlin/2019/06/kotlin-1-3-40-released/): https://plugins.gradle.org/plugin/org.jetbrains.kotlin.js

Our plugin should add support for this plugin as well.

### PR DESCRIPTION
**Pull request by Tapchicoma**
Closes #252

### CODE REVIEW

File: plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/KotlinJsPluginTests.kt [hunk 1]
```diff
@@ -0,0 +1,73 @@
+package org.jlleitschuh.gradle.ktlint
```
Comment (JLLeitschuh) [inline plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/KotlinJsPluginTests.kt]: Thanks for staying on top of these compatibility tests and keeping them refactored as they need to be changed.

File: plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/KotlinJsPluginTests.kt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+import org.assertj.core.api.Assertions.assertThat
```
Comment (JLLeitschuh) [inline plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/KotlinJsPluginTests.kt]: This is a nice API that `assertThat` proivides. I may have to give it a try.

File: samples/kotlin-kotlin2js/src/main/kotlin/org/jlleitschuh/gradle/ktlint/sample/js/main.kt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package org.jlleitschuh.gradle.ktlint.sample.js
+
+import kotlin.browser.document
+
+val secondDiv: dynamic = document.createElement("div").apply {
```
Comment (JLLeitschuh) [inline samples/kotlin-kotlin2js/src/main/kotlin/org/jlleitschuh/gradle/ktlint/sample/js/main.kt]: Forgive my lack of experience. What is the `dynamic` type in Kotlin? This is the first I've ever seen it.

Comment (Tapchicoma) [reply samples/kotlin-kotlin2js/src/main/kotlin/org/jlleitschuh/gradle/ktlint/sample/js/main.kt]: It is only js specific: https://kotlinlang.org/docs/reference/dynamic-type.html

**Pull request merged**